LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2018 birmachera
```

Comment:
use this licence file: https://github.com/bitrise-steplib/step-template/blob/master/LICENSE